src/sentry/event_manager.py

```diff
@@ -881,6 +881,13 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                 "group.created", skip_internal=True, tags={"platform": event.platform or "unknown"}
             )
 
+            group_created.send_robust(
+                sender=self,
+                group=group,
+                project=event.project,
+                organization=event.project.organization,
```

Comment:
let's make sure this won't make additional queries (I know django does lazy loading of objects on-demand in some situations). in fact, i may not be the best person to sign off on event_manager.py changes, so could you get another pair of eyes on this?

Reply:
(everything else looks fine to me)

Reply:
I am using `event` to access those objects, they look like they are loaded from the cache:
`project`: https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L521
`project.organization`: https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L492